docs/resources/Webhook.md

```diff
@@ -193,6 +193,7 @@ Refer to [Uploading Files](#DOCS_REFERENCE/uploading-files) for details on attac
 | payload_json \*\*| string                                                                               | JSON encoded body of non-file params                                                                                                                                        | `multipart/form-data` only   |
 | attachments \*\* | array of partial [attachment](#DOCS_RESOURCES_CHANNEL/attachment-object) objects     | attachment objects with filename and description                                                                                                                            | false                        |
 | flags            | integer                                                                              | [message flags](#DOCS_RESOURCES_CHANNEL/message-object-message-flags) combined as a [bitfield](https://en.wikipedia.org/wiki/Bit_field) (only `SUPPRESS_EMBEDS` can be set) | false                        |
+| thread_name            | string                                                                              | name of thread to create (only available if webhook is in a forum channel) | false                        |
```

Comment:
I think it needs a little bit more info here. On line 171 there's an info block already for message content, so maybe add to that or make a second one, but call out then when running the webhook in a forum, one of thread_name/thread_id is also required, the former creating a new thread, the latter posting into an existing thread.